test/test_evaluator.cpp

```diff
@@ -367,20 +367,10 @@ TEST_F(ExprEvalTest, TensorMetaData) {
 
 TEST_F(ExprEvalTest, OpaqueEquality) {
   Opaque a{DataType::Int}, b{DataType::Int};
```

Comment:
Are these initialized to some value by default?

Reply:
Do you mean `Opaque`'s default initializer? I don't think it have one.

Reply:
Ah, I somehow misunderstood `Opaque` is supposed to contain a value of a given type, but in this case it actually works like a polymorphic type, IIUC. So they are initialized to `DataType::Int` as their values.